client/performance-profiler/components/metrics-insight/insight-content.tsx

```diff
@@ -31,7 +31,7 @@ export const InsightContent: React.FC< InsightContentProps > = ( props ) => {
 	return (
 		<div className="metrics-insight-content">
 			{ isLoading ? (
-				<LLMMessage message={ translate( 'Finding the best solution…' ) } rotate />
+				<LLMMessage message={ translate( 'Finding the best solution for your site…' ) } rotate />
```

Comment:
If this new string isnt translated yet, you could support other locales in the meantime by using a temporary `hasEnTranslation` check to determine whether to serve the new (...for your site) translation or not.